TSheetsTeam/api_docs
# Updating Getting Help resource links

Updating project to use podman instead of docker
Updating 'Getting Help' resource links

To test:
Go to the 'Getting Help' section
Verify the wording is good
Verify the links work
Click around on a few parts of the API docs and ensure everything works
